test/e2e/framework/util.go

```diff
@@ -4080,3 +4080,19 @@ func UpdatePodWithRetries(client *client.Client, ns, name string, update func(*a
 	}
 	return nil, fmt.Errorf("Too many retries updating Pod %q", name)
 }
+
+func GetPodsInNamespace(c *client.Client, ns string, ignoreLabels map[string]string) ([]*api.Pod, error) {
```

Comment:
I'll do that in a followup PR, so that we can get this in sooner (it has passed tests already).